packages/ember/tsconfig.json

```diff
@@ -0,0 +1,56 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "allowJs": true,
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true,
+    "noFallthroughCasesInSwitch": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noEmitOnError": false,
+    "noEmit": true,
+    "inlineSourceMap": true,
+    "inlineSources": true,
+    "baseUrl": ".",
+    "module": "es6",
+    "experimentalDecorators": true,
```

Comment:
Can we somehow reduce this and align it with other `tsconfig.json` we have in other packages.
This file should really only contain what absolutely necessary and ideally, extend our root `tsconfig.json` like other packages.

Reply:
I'll take a look 👍

Reply:
I've extended and reduced it like you mentioned, but again Ember is conventional with which typescript it uses (modules are aimed at es6 and decorators are strongly encouraged to use in Octane). Guidance around settings in Ember is to usually leave defaults unless you have a very specific reason to change them, and rely on tooling (automated cli tools, codemods, etc) to upgrade and update.

Reply:
Makes sense. What version of Ember will this support then?

Reply:
All supported LTS versions. They are tested with ember-try (see here: https://github.com/getsentry/sentry-javascript/pull/2739/files#diff-7545849fb3cdafe10c02f667d5efe2f0), I'll add a note to the README though.